### PR TITLE
Reduce logging for functions invocations

### PR DESCRIPTION
**Pull request by vrdmr**
## Description
Reduce logging for functions invocations to optimize for performance. This reduces the # of log statements from 3 to 1 (merging the first 2 log lines and removing the successful invocation)

---

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azure_functions_worker/dispatcher.py [hunk 1]
```diff
@@ -7,37 +7,35 @@
...
+from .logging import disable_console_logging, enable_console_logging
+from .logging import error_logger, is_system_log_category, logger
 from .utils.common import get_app_setting
 from .utils.tracing import marshall_exception_trace
 from .utils.wrappers import disable_feature_by
-from asyncio import BaseEventLoop
-from logging import LogRecord
-from typing import Optional
+
+_TRUE = "true"
```
Comment (Hazhzeng) [inline azure_functions_worker/dispatcher.py:38]: Put it in constants.py would be a better place.

**Review by Hazhzeng — APPROVED** [review]
Thanks for cleaning this up, neat!

File: azure_functions_worker/dispatcher.py (hunk 1 above)
Comment (vrdmr) [reply azure_functions_worker/dispatcher.py:38]: Constants.py seem to be specific to more config-level or global setting and keeping constants within the modules they are used is a better approach.

If the constant is to be used in multiple modules, constants.py is better and we can refactor this when we see it being used in other places.

Comment (Hazhzeng) [reply azure_functions_worker/dispatcher.py:38]: Ah, I see, thanks for the explanation!

**Pull request merged**